cross/pango/Makefile

```diff
@@ -0,0 +1,23 @@
+PKG_NAME = pango
+PKG_VERS = 1.28.4
+PKG_SHORT_VERS = 1.28
```

Comment:
Please avoid copy-paste. Reuse `PKG_SHORT_VERS` in `PKG_VERS` value.

Reply:
I remember, we had this once before...